Dockerfile

```diff
@@ -35,12 +35,12 @@ RUN apk update \
     && apk --purge del .build-deps
 
 # install app package source, avoid copying the rest
-COPY ./bin $MAGPIE_DIR/bin/
 COPY ./config/magpie.ini $MAGPIE_CONFIG_DIR/magpie.ini
 COPY ./env/*.env.example $MAGPIE_ENV_DIR/
 COPY ./magpie $MAGPIE_DIR/magpie/
 # equivalent of `make install` without conda env and pre-installed packages
 RUN pip install --no-dependencies -e $MAGPIE_DIR
 
 # equivalent of `make cron start` without conda env
-CMD crond -c $CRON_DIR && gunicorn -b 0.0.0.0:2001 --paste $MAGPIE_CONFIG_DIR/magpie.ini --workers 10 --preload
+# bind to '0.0.0.0' such that any IP employed by the server will retrieve the application endpoint by default
+CMD crond -c $CRON_DIR && pserve "$MAGPIE_CONFIG_DIR/magpie.ini"
```

Comment:
Number of workers was override to 10 here while set to 3 in the ini file.

Reply:
It should now be done in the INI. It didn't make much sense to be in the command in the first place. 
It is very hard to track that this value was overridden here, and the first impression is usally that whatever was in the INI is the 'truth'.

Reply:
Yeah, I'm fine with the fact that the INI is the 'truth'. I'm more concerned about the fact that we go from 10 to 3 workers. I would not have said a word if the INI set workers to 10 too.

Reply:
I can set it by default to 10, so that it will be transparent for anyone that used the image without override of the command.

Reply:
Yes that's the kind of backward compatibility that we must keep in mind and avoid unnecessary changes when possible.